crawl-ref/docs/develop/background_creation.txt

```diff
@@ -85,7 +85,7 @@ herd-watcher:
 
         // and just a couple skill points in stuff you might do around a field.
         { { SK_FIGHTING, 6 }, { SK_SUMMONING, 8 }, { SK_SPELLCASTING, 3 },
-          { SK_DODGING, 3 }, { SK_ARMOR, 4 } { SK_STEALTH, 1 }, },
+          { SK_DODGING, 3 }, { SK_ARMOUR, 4 } { SK_STEALTH, 1 }, },
```

Comment:
despite prev comment about developer docs, this one doesn't match the code so should be changed